app/Models/CovidEntry.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class CovidEntry extends Model
+{
+    use HasFactory;
+
+    protected $table = 'area As a';
```

Comment:
Why `CovidEntry` table name like this?. Pls check laravel `Eloquent`

Reply:
I will change it.